src/picknik_ur_base_config/objectives/request_teleoperation.xml

```diff
@@ -26,7 +26,7 @@
             <Decorator ID="ForceSuccess" _while="teleop_mode == 4">
               <Control ID="Sequence">
                 <Action ID="SwitchUIPrimaryView" primary_view_name="Visualization"/>
-                <SubTree ID="Move to Pose"/>
+                <SubTree ID="Move to Pose" controller_names="/joint_trajectory_controller"/>
```

Comment:
Is the hardcoded controller name necessary?

Reply:
I made `controller_names` as a "param" for `Move to Pose` subtree because this subtree requires different controllers in other cases

Reply:
That makes sense, thanks!